src/lib/mina.ts

```diff
@@ -1239,15 +1254,20 @@ async function verifyAccountUpdate(
         publicOutput: [],
       };
 
-      let verificationKey = account.zkapp?.verificationKey?.data!;
+      let verificationKey = account.zkapp?.verificationKey?.data;
+      assert(
+        verificationKey !== undefined,
+        'Account does not have a verification key'
+      );
```

Comment:
ran into this being `undefined` in a test and failed down the line - better to have an explicit error about it right away